test/test_worker.py

```diff
@@ -24,6 +24,24 @@ def test_basic(self):
             'stalled': 0
         }])
 
+    def test_worker_registration_lapse(self):
+        '''Workers that fail to check in expire from the list of active workers'''
+        self.lua('config.set', 0, 'max-worker-age', 10)
+        self.lua('pop', 1, 'queue', 'worker', 1)
+        self.assertEqual(self.lua('workers', 15), {})
```

Comment:
I would expect this to assert against `[{}]` based on the other test. Is this a lua quirk?

Reply:
Yeah -- Lua has a data structure `tables` that is both an array and an associative data structure. When serializing an empty table to JSON, it comes back as `{}` instead of `[]`